packages/react-scripts/package.json

```diff
@@ -78,8 +78,8 @@
     "workbox-webpack-plugin": "^6.4.1"
   },
   "devDependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0"
```

Comment:
kind of surprise we could update this so easily. We probably have a lot of `act` warnings now?

Reply:
This won't affect any of our existing use cases of the fork, it's just a dev dependency for react-scripts. But we could start going through to update to React 18 in our apps to find out how hard it will be to update...